guardian/support-service-lambdas
# Zuora <-> Salesforce link remover - restucture folders to fix build error

## What does this change?
Fix folder structure and import references so fix error on lambda execution:

![image](https://github.com/guardian/support-service-lambdas/assets/36296660/2e01f774-838d-4fda-9b42-4f1c409843d0)

Thanks to @AndreaDiotallevi for helping to diagnose and fix the issue

## How to test
- execute the state machine and verify each lambda runs without error
![image](https://github.com/guardian/support-service-lambdas/assets/36296660/c7f8a80c-10b2-4ec0-a6d3-895cb76e0e0b)
